scripts/variantstore/variant_annotations_table/README.md

```diff
@@ -0,0 +1,111 @@
+# Creating the Variant Annotations Table
+
+### The VAT pipeline is a set of WDLs
+- [GvsCreateVAT.wdl](scripts/variantstore/wdl/GvsCreateVAT.wdl)
+- [GvsValidateVAT.wdl](scripts/variantstore/variant_annotations_table/GvsValidateVAT.wdl)
+
+The pipeline takes in a jointVCF and outputs a table in BigQuery.
+
+**GvsCreateVAT** creates the table, while
+**GvsValidateVAT** checks and validates the VAT
+
+
+### Run GvsCreateVAT:
+
+Most of the inputs are constants — like the reference, or a table schema — and dont require any additional work (and paths can be found in the [example inputs json](scripts/variantstore/wdl/GvsCreateVAT.example.inputs.json)). However for the specific data being put in the VAT, three inputs need to be created.
```

Comment:
Since we've been talking about this in refinement...etc. now might be a good time to either strip out inputs that can be hard-coded (maybe `table_suffix`) or just add as variables like `contig_array`.

Reply:
I want users to be able to name their VAT table...at least I thought I did....will users not want that?

Reply:
but for the reference I can hardcode it, and for the items in my scratch bucket, I feel weird about it, but I can hardcode those for now too